selfdrive/car/toyota/carstate.py

```diff
@@ -140,7 +140,7 @@ def get_can_parser(CP):
       ("STEERING_LEVERS", 0.15),
       ("SEATS_DOORS", 3),
       ("ESP_CONTROL", 3),
-      ("EPS_STATUS", 25),
+      ("EPS_STATUS", 0),
```

Comment:
Parser is saying it never got this signal from logs, but it exists on Cabana under bus 0